torch/autograd/profiler.py

```diff
@@ -414,8 +465,9 @@ def parse_nvprof_trace(path):
         unique.see(row['marker_id'])
         evt = FunctionEvent(id=row['marker_id'],
                             name=strings[row['name']],
-                            start=row['start_time'],
-                            end=row['end_time'])
+                            cpu_interval=Interval(row['start_time'], row['end_time']),
+                            cuda_interval=None,
+                            thread=0)
```

Comment:
You can get a thread id from these tables. Just look for an appropriate column (it's an sqlite db so you can open it in terminal)